packages/utils/parcel-lsp/package.json

```diff
@@ -28,10 +28,10 @@
     "parcel": "^2.7.0"
   },
   "dependencies": {
-    "@parcel/diagnostic": "2.5.0",
-    "@parcel/plugin": "2.5.0",
-    "@parcel/types": "2.5.0",
-    "@parcel/utils": "2.5.0",
```

Comment:
Were these intentionally set to use old versions of parcel?

If we are interested in syncing all packages in the repo against their declared workspace version, [pnpm](https://pnpm.io/workspaces) has phenomenal workspace support that avoids phantom dependencies and allows us to describe package dependencies with:
```json
"dependencies": {
    "@parcel/types": "workspace/*"
}
```
which automatically transforms the version to the version declared in the workspace package on package publishing.
```json
"dependencies": {
    "@parcel/types": "2.7.0"
}
```
If a dependency with a different version is required, it installs it locally to the package without creating conflicts with other packages

Reply:
> Were these intentionally set to use old versions of parcel?

No, but for some reason this step didn't bump the version on the last release
https://github.com/parcel-bundler/parcel/blob/932b6bd702b0e6af573713840cbb47cd3afd6e17/package.json#L33

> `"@parcel/types": "workspace/*"`

Is there some mechanism to replace that with the actual version on publish? Or is that only meant for private packages?

Reply:
> Is there some mechanism to replace that with the actual version on publish? Or is that only meant for private packages?

It's a mechanism to replace the version with the actual version when running `pnpm pack` or `pnpm publish` - it works for public or private packages as it's just string replacement when publishing.

Reduces the maintenance burden involved in tracking the versions of packages in the workspace and avoids us needing to commit absolute versions in the source repo which has caused me so much grief in the past 🤣

To be clear, this is a [feature of pnpm workspaces](https://pnpm.io/workspaces#publishing-workspace-packages) and I don't believe it's available in lerna or yarn - so it's currently unavailable to us, but is that something we'd be open to discussing?